custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -193,3 +201,77 @@ def _dump_consolidated_data(result):
             row['State'] = state_name
             writer.writerow(row)
         return result_file
+
+
+class VHSNDMonthlyReport(MonthBasedDataPull):
+    slug = "vhsnd_monthly_report"
+    name = "VHSND monthly report"
+    queries = [
+        VHSNDMonthlyCount
+    ]
+    file_format_required = 'xlsx'
+
+    def post_run(self, data_files):
+        state_results = self._consolidate_data(data_files)
+        return self._dump_consolidated_data(state_results)
+
+    def _consolidate_data(self, data_files):
+        result = defaultdict(dict)
+        state_name_column = 'state_name'
```

Comment:
what's the need of this variable if this is used only at one place?

Reply:
Same for `vhnsnd_date_column`

Reply:
changed it